src/interfacers/EmonHubMinimalModbusInterfacer.py

```diff
@@ -66,6 +66,24 @@
         names =  V1,V2,V3,I1,I2,I3,P1,P2,P3,TotalPower,Import_kWh,Export_kWh
         precision = 2,2,2,2,2,2,2,2,2,2,2,2
 
+[[SONTEX_EIA485]]
```

Comment:
This is just my setup verbatim, it may not be suitable as a generic config example.